torch/_dynamo/utils.py

```diff
@@ -252,7 +252,13 @@ def dynamo_timed(
     try:
         with torch.profiler.record_function(f"{key} (dynamo_timed)"):
             t0 = time.time()
+            ChromiumEventLogger.log_event_start(key, time.time_ns())
+            if phase_name:
```

Comment:
I'm not actually sure what event we want to emit here when there's a phase and a function name, so I just log both. But open to suggestions.

Reply:
Some history on dynamo_timed, in the old days, key was inferred directly from `func.__name__` of the function we wrapped over. However, later, people wanted to put some metrics into the scuba table, and a pile of random internal function names would not have been great, so then they added a phase to attribute things dynamo/aotautograd/inductor/codegen. Then recently, I un-decorator'ified this function, so now name is explicitly passed in.

Only one dynamo_timed ever gets a phase name. I think it's OK to just override the not as useful func with phase name, instead of duping the events.